test/Digest.test.ts

```diff
@@ -0,0 +1,14 @@
+import {Digest} from "../dist/Digest";
+
+import {describe, test, expect} from "@jest/globals";
+
+describe("Digest", () => {
+    describe("Digest.getAuthHeader", () => {
```

Comment:
tady bych mozna uz neopakoval jmeno tridy kdyz je hned o radek vyse..ono by to pak v tom testu melo byt videt oboje kdyz to failne..

Reply:
!